examples/pulsar-operators/proxy.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
   name: zookeepers
   namespace: pulsar
 spec:
-  image: streamnative/sn-platform-slim:2.10.3.4
+  streamnative/sn-platform-slim:2.10.5.3
```

Comment:
This is broken in `proxy.yaml`, `storage.yaml` and `quick-start.yaml` @ericsyh @urfreespace

Reply:
@z0mbix thx a lot! I will push a PR to fix.

Reply:
will fix by https://github.com/streamnative/charts/pull/1108